libraries/Microsoft.Bot.Builder.Prompts/Choices/ChoiceFactory.cs

```diff
@@ -66,14 +66,14 @@ public static IMessageActivity ForChannel(string channelId, List<Choice> list, s
             }
         }
 
-        public static Activity Inline(List<string> choices, string text = null, string speak = null, ChoiceFactoryOptions options = null)
+        public static Activity Inline(IEnumerable<string> choices, string text = null, string speak = null, ChoiceFactoryOptions options = null)
         {
             return Inline(ToChoices(choices), text, speak, options);
         }
 
-        public static Activity Inline(List<Choice> choices, string text = null, string speak = null, ChoiceFactoryOptions options = null)
+        public static Activity Inline(IEnumerable<Choice> choiceList, string text = null, string speak = null, ChoiceFactoryOptions options = null)
```

Comment:
`choiceList` doesn't seem as good a name as `choices` here, especially now that it's not an actual `List<T>`... did you mean to change???

Reply:
I'll reverse the names so that the parameter keeps the name **choices** and the introduced local variable becomes **choiceList**, here and in the other places where I made the same change.